.github/workflows/lint.yaml

```diff
@@ -67,6 +67,9 @@ jobs:
           level: warning
           filter_mode: diff_context
           glob_pattern: "**/*.py"
+      - name: Install specified mypy version
+        run: |
+          python -m pip install -q -r requirements-dev.txt
```

Comment:
I chose requirements-dev.txt instead of `pip install mypy==0.782` so we don't have to update a lot of hardcoded versions when upgrading in the future.